apis/.schemes/config-v1alpha1-LandscaperConfiguration.json

```diff
@@ -47,6 +47,76 @@
         }
       }
     },
+    "config-v1alpha1-CommonControllerConfig": {
+      "description": "CommonControllerConfig describes common controller configuration that can be included in the specific controller configurations.",
+      "type": "object",
+      "required": [
+        "workers",
+        "cacheSyncTimeout"
+      ],
+      "properties": {
+        "cacheSyncTimeout": {
+          "description": "CacheSyncTimeout refers to the time limit set to wait for syncing caches. Defaults to 2 minutes if not set.",
```

Comment:
Is somewhere described what is cached?

Reply:
its the controller's watch/resources cache.
I thought that it is clear because its `controllers.installations.cacheSyncTimeout`.
But I can add comment to make it clearer